math/fastor/include/algebra/math/impl/fastor_vector.hpp

```diff
@@ -0,0 +1,57 @@
+/** Algebra plugins, part of the ACTS project
+ *
+ * (c) 2020-2022 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+// Project include(s).
+#include "algebra/math/common.hpp"
```

Comment:
Do you need this include?

Reply:
I'm actually not 100% sure. The other backends had it so I put it in for Fastor as well. From a cursory glance, it looks like that file decides where to pick up math functions from.

Reply:
Hm, I don't think you are using ```sqrt``` fot the norm, so you don't need it, I would expect. SMatrix and Eigen, who have a dedicated normalize function, also don't include it.

Reply:
You're right. I'll remove it.